services/auth-service/adapter-kafka/src/main/java/com/kjeldsen/auth/kafka/events/Event.java

```diff
@@ -0,0 +1,11 @@
+package com.kjeldsen.auth.kafka.events;
+
+import lombok.experimental.SuperBuilder;
+
+import java.time.Instant;
+
+@SuperBuilder
+public abstract class Event {
```

Comment:
nice! Would you like to try and create a library for all common events stuff? just checking how I did with the sec lib. Only if you are interested <3 if not we are good like this

Reply:
Yes, if there is possibility I would like to do it, it seems useful.